### ISSUE TITLE
access to kerberized data

### ISSUE THREAD

**Issue by vder** [trinodb/grafana-trino#254]
Hi, can I use trino plugin to acess data that requires kerberos authentication? At first glance it rather wont work, bu tmaybe i missed something?

**Comment by nineinchnick** [comment]
I haven't tested it with Kerberos, but I don't think it'll work out of the box. The Trino Go driver also doesn't have any Kerberos support.

**Comment by vder** [comment]
Thanks.

**Comment by vder** [comment]
Hi I took a look into the code and as far I understand it kerberos connections are supported in trino go client, so If I wanna add this feature to plugin, all I have to do is add some fields for new parameters and then pass them to trino client, right?

**Comment by nineinchnick** [comment]
Yes, hopefully, you should be able to find existing examples in Grafana docs how Kerberos authentication is usually handled. I know they describe this at least for OAuth2.

**Comment by youaodu** [comment]
hello, what method was used in the end for this issue?

**Comment by nineinchnick** [comment]
This is not resolved yet.
